easybuild/easyconfigs/g/GLib/GLib-2.48.0-foss-2016a.eb

```diff
@@ -21,7 +21,8 @@ dependencies = [
 
 builddependencies = [('Python', '2.7.11')]
 
-configopts = "--disable-maintainer-mode --disable-silent-rules --disable-libelf --enable-static --enable-shared --disable-systemtap"
+configopts = "--disable-maintainer-mode --disable-silent-rules --disable-libelf --enable-static --enable-shared "
+configopts += "--disable-systemtap"
```

Comment:
but this `--disable` with the others, and move the `--enable-*` ones to a new line instead?